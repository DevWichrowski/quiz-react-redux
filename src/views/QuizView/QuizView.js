import React from 'react';
import Layout from "../../components/shared/Layout/Layout";
import Quiz from "../../components/Quiz/Quiz";

const QuizView = () => {
    return (
        <div>
            <Layout>
                <Quiz/>
            </Layout>
        </div>
    );
};

export default QuizView;