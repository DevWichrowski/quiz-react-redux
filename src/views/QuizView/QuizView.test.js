import {testComponentRender} from "../../core/helpers/tests/test-component-render";
import QuizView from "./QuizView";
import React from "react";


describe('QuizView', () => {
    testComponentRender(<QuizView/>)
});
