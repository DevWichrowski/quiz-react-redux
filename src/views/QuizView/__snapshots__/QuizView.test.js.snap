// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizView should render 1`] = `
<Layout>
  <Connect(Quiz) />
</Layout>
`;
